backend/settings.py

```diff
@@ -26,25 +28,18 @@
 SECRET_KEY = os.environ.get('DJANGO_SECRET_KEY', '=*f9))56@c*dzlmm0%t@)v1=)d2*pbom51h+o7l%35xt92ya3t')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = os.environ.get('DEBUG') == 'true'
+# DEBUG = not _isProd
+#TODO don't commit this!
+DEBUG = True
 
 # URLs used to talk to the backend
-ALLOWED_HOSTS = [
-    'testserver',
-    'localhost',
-    '127.0.0.1',
```

Comment:
idk if this is great because some people may prefer running the app without docker compose and running each component individually. I know I do because it lets me use visual debugging in pycharm.

Reply:
I see.. I don't have context on that but I assume you'll need localhost? 127.0.0.1 as well?

You may run into some SSL issues running without docker (it's certainly do-able to set up but I didn't account for that)

Is it possible to somehow enable visual debugging with docker?

Reply:
I've added localhost LMK if that's sufficient

Reply:
I haven't figured out how to attach a debugger to a docker container but there might be a way. Localhost is good for now. I'll figure out how to get debugging working in pycharm.